modules/deploy/pages/deployment-option/self-hosted/manual/sizing.adoc

```diff
@@ -14,12 +14,22 @@ Throughput and retention requirements can cause bottlenecks in the system. On an
 
 In general, choose the number of nodes based on the following criteria, and add nodes for fault tolerance. This ensures that the system can operate with full throughput in a degraded state.
 
-* Total network bandwidth must account for maximum simultaneous writes and reads, multiplied by the replication factor.
-* Have a minimum of 2 GB memory for each core, but more memory is better.
-* Have a minimum of 4 MB of memory for each topic partition replica. You can enforce this requirement in the tunable xref:reference:tunable-properties.adoc#topic_memory_per_partition[`topic_memory_per_partition` property].
-* Use Tiered Storage to unify historical and real-time data.
-* For AWS, test with either i3en, i4i, or is4gen; for GCP, test with n2-standard (GCP); and for Azure, test with lsv2. In general, choose instance types that prioritize storage and network performance.
-* Run hardware and Redpanda benchmark tests for a performance baseline.
+* **Network bandwidth**: Total bandwidth must account for maximum simultaneous writes and reads, multiplied by the replication factor.
+* **Memory per core**: Allocate a minimum of 2 GB memory for each CPU core. More memory improves performance.
+* **Memory per partition**: Allocate a minimum of 2 MB of memory for each topic partition replica. 
++
+For example: If you have 10,000 total partitions with a replication factor of 3, you need at least 60 GB of memory across all brokers:
++
+10,000 partitions × 3 replicas × 2 MB = 60,000 MB (60 GB)
+* **Storage strategy**: Use Tiered Storage to unify historical and real-time data cost-effectively.
+* **Performance testing**: Run hardware and Redpanda benchmark tests to establish a performance baseline. Choose instance types that prioritize storage and network performance:
++
+--
+* **AWS**: Test with i3en (NVMe SSD), i4i (latest NVMe), or is4gen (Intel-based NVMe)
+* **Azure**: Test with Lsv2-series (high I/O performance)
+* **GCP**: Test with n2-standard instances with local SSD
```

Comment:
These don't reflect our current guidelines nor our actual instance types used in cloud. I have an internal page here which should be close to the truth:

https://redpandadata.atlassian.net/wiki/spaces/CORE/pages/1298792513/Instance+selection

H/e, I guess I can file a separate issue for it...

Reply:
Thanks @travisdowns: please file a separate issue. As part of reviewing/updating all the content, we could also do a style clean-up to make the content easier to read.